src/components/LoginOptions/index.js

```diff
@@ -12,8 +12,9 @@ import LoginOptionsDisplay from './LoginOptionsDisplay';
 const LoginOptions = ({form, onFormStart}) => {
   const config = useContext(ConfigContext);
 
-  const loginAsYourselfOptions = [];
-  const loginAsGemachtigdeOptions = [];
+  let loginAsYourselfOptions = [];
```

Comment:
Why do we change this to `let` ?I thought that `const` is ok with arrays even if we modify the content (the refernece is still to `loginAsYourselfOptions`)